Examples/Complete/PointCloudPotree2/ImGui/PointCloudControlCore.cs

```diff
@@ -87,7 +87,7 @@ public override void Init()
                 };
 
                 var potreeReader = new Potree2Reader();
-                _pointCloud = (PointCloudComponent)potreeReader.GetPointCloudComponent(PtRenderingParams.Instance.PathToOocFile);
+                _pointCloud = (PointCloudComponent)potreeReader.GetPointCloudComponent(PtRenderingParams.Instance.PathToOocFile, false);
```

Comment:
I'm always uncertain if one should convert every `bool` to an `enum` for readability.

Reply:
Why not, maybe we'll add another point cloud `RenderMode` - e.g using Compute Shaders.